components/CreateFlow.js

```diff
@@ -0,0 +1,140 @@
+import { nanoid } from "nanoid";
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function CreateFlowForm({ flows, setFlows, setOpen }) {
+  const [hoursValidation, setHoursValidation] = useState(0);
```

Comment:
Why not call the state hours? It was confusing to me that you called it hoursValidation. A variable name should always be descriptive of the value that it tracks. You can then use the variable for anything you want, e.g. validation or some calculation.

Reply:
True, that is much more understandable. I changed it.